extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -166,6 +168,17 @@ class CornerstoneViewportService implements IViewportService {
     const viewportInfo = this.viewportsInfo.get(viewportIndex);
     viewportInfo.setRenderingEngineId(renderingEngine.id);
 
+    const priorViewportOptions = viewportInfo.getViewportOptions();
+    const viewport = renderingEngine.getViewport(viewportInfo.getViewportId());
+    if (viewport) {
```

Comment:
This stuff just remembers the data - if no applier is found, then nothing new happens, but it at least allows remembering the viewport data.